libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -74,4 +76,36 @@ export class StripeManager {
       customer.tax?.automatic_tax === 'not_collecting'
     );
   }
+
+  /**
+   * If customer taxId does not match incoming taxId, update customer record with incoming taxId.
+   *
+   * @param customerId
+   * @param taxId
+   * @returns
+   */
+  async addTaxIdToCustomer(customerId: string, taxId: string) {
+    // fetch customer by customerId
+    const customer = await this.client.fetchCustomer(customerId);
+
+    // check that customer exists
+    if (!customer) {
+      throw new Error('customer does not exist');
+    }
+
+    // check that customer hasn't been deleted
```

Comment:
[non-blocking nit] This comment doesn't really add value, since the code below says the same thing as this comment

Reply:
In general I'd apply that to each of the comments in the method. Would suggest adding comments only if they're needed to describe/explain the behavior of the code

Reply:
Removed. See reply above for explanation. 🙏 Thank you!